client/app/bundles/course/statistics/pages/StatisticsIndex/index.jsx

```diff
@@ -77,9 +90,25 @@ const StatisticsIndex = ({
   courseStatistics,
   studentsStatistics,
   staffStatistics,
-  intl,
 }) => {
   const [value, setValue] = useState(0);
+  const intl = useIntl();
+
+  useEffect(() => {
```

Comment:
I now think if the data should be fetched separately for different tabs. Otherwise, we may make unnecessary load to the backend. Then when switching between tabs, we can check if the data already exist in the redux, we dont make any further API calls.